packages/modules/delete-itwin/LICENSE.md

```diff
@@ -0,0 +1,9 @@
+# MIT License
+
+Copyright © 2017-2023 Bentley Systems, Incorporated. All rights reserved.
```

Comment:
```suggestion
Copyright © Bentley Systems, Incorporated. All rights reserved.
```

Reply:
I copied what was in itwinjs-core

Reply:
most of those pkgs have been published since 2017
afaik new pkgs shouldnt include the yr until the new yr after and it needs to be updated

Reply:
Ok makes sense. I shouldn't have copied from itwinjs-core because this package wasnt published when itwinjs-core was.